docs/fundamentals/code-analysis/quality-rules/ca1422.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "CA1422: Validate platform compatibility"
+description: "Learn about code analysis rule CA1422: Validate platform compatibility"
+ms.date: 11/03/2022
+ms.topic: reference
+f1_keywords:
+- CA1422
+helpviewer_keywords:
+- CA1422
+dev_langs:
+- CSharp
+- VB
+---
+# CA1422: Validate platform compatibility
```

Comment:
NIT: I feel like the title same as the CA1416 might be confusing, I might have had changed the title for this a bit, can we change it in the doc at least? Something like
```suggestion
# CA1422: Validate platform compatibility - obsoleted APIs
```